src/content/documentation/publish/source-code-submission.md

```diff
@@ -78,7 +78,7 @@ Tip: Use a build target relative to the directory containing the source, such as
 If you need to provide it, matching source code must be attached to every extension version.
 :::
 
-Code submission process happens after the add-on validation step.
+Code submission process happens after the add-on validation step. The maximum file size for source code files is 200MB.
```

Comment:
Per @wagnerand's [suggestion](https://github.com/mozilla/extension-workshop/pull/1883#issuecomment-2154676465), could you also note "the limit on the uncompressed file size as well?"

```suggestion
Code submission process happens after the add-on validation step. The maximum compressed file size for source code files is 200MB and ???MB uncompressed.
```

Reply:
I thought this was already the uncompressed max size as per v